CHANGELOG.md

```diff
@@ -1,3 +1,191 @@
+    [here](https://github.com/weaveworks/scope/tree/master/examples/plugins).
+  * Metrics on canvas: Metrics are now displayed on the nodes and not just on
+    the details panel, starting with CPU and memory consumption.
+
+Also, the performance of the UI has been considerably improved and the 100-node
+rendering limit has been lifted.
+
+
+New features and enhancements:
+- Metrics on canvas
+	[#1105](https://github.com/weaveworks/scope/pull/1105)
+	[#1204](https://github.com/weaveworks/scope/pull/1204)
+	[#1225](https://github.com/weaveworks/scope/pull/1225)
+- Scope Plugins
+	[#1126](https://github.com/weaveworks/scope/pull/1126)
+	[#1277](https://github.com/weaveworks/scope/pull/1277)
+	[#1280](https://github.com/weaveworks/scope/pull/1280)
+	[#1283](https://github.com/weaveworks/scope/pull/1283)
+- Node details panel improvements
+  - Add connection tables
+	[#1017](https://github.com/weaveworks/scope/pull/1017)
+	[#1248](https://github.com/weaveworks/scope/pull/1248)
+  - Layout: make better use of column space
+		[#1272](https://github.com/weaveworks/scope/pull/1272)
+  - Sparklines
+    - Update every second and show 60sec history
+			[#795](https://github.com/weaveworks/scope/pull/795)
+    - Apply format to tooltips in hovers
+			[#1230](https://github.com/weaveworks/scope/pull/1230)
+  - Sort numerical entries (e.g. image counts, process IDs) as expected
+		[#1125](https://github.com/weaveworks/scope/pull/1125)
+  - Remove load5 and load15 metrics
+		[#1274](https://github.com/weaveworks/scope/pull/1274)
+- Graph view improvements
+  - Node filtering improvements
+      - Introduce three-way filtering selectors (e.g. choose from _System containers_, _Application containers_ or _Both_)
+			[#1159](https://github.com/weaveworks/scope/pull/1159)
+      - Maintain node-filtering selection across subviews (e.g. _Containers by ID_ and _Containers by Image_)
+			[#1237](https://github.com/weaveworks/scope/pull/1237)
+  - Refine maximum length of node names
+		[#1263](https://github.com/weaveworks/scope/issues/1263)
+		[#1255](https://github.com/weaveworks/scope/pull/1255)
+  - Refine border-width of nodes
+		[#1138](https://github.com/weaveworks/scope/pull/1138)
+		[#1120](https://github.com/weaveworks/scope/pull/1120)
+- Enable launching terminals in hosts
+	[#1208](https://github.com/weaveworks/scope/pull/1208)
+- Allow pausing the UI through a button
+	[#1106](https://github.com/weaveworks/scope/pull/1106)
+- Split the internet node for incoming vs outgoing connections.
+	[#566](https://github.com/weaveworks/scope/pull/566)
+- Show k8s pod status
+	[#1289](https://github.com/weaveworks/scope/pull/1289)
+- Allow customizing Scope's hostname in Weave Net with `scope launch --weave.hostname`
+	[#1041](https://github.com/weaveworks/scope/pull/1041)
+- Rename `--weave.router.addr` to `--weave.addr` in the probe for consistency with the app
+	[#1060](https://github.com/weaveworks/scope/issues/1060)
+- Add `/api/probes` endpoint
```

Comment:
Or potentially introduce another section, and add multitenancy to that section too?